app/controllers/api/current/appointments_controller.rb

```diff
@@ -46,6 +46,7 @@ def appointments_params
         :cancel_reason,
```

Comment:
Can we add an explicit check to see that the API marks appointment_type

Reply:
Will add some relevant specs for the same.

Reply:
Added specs for `appointment_type` checks against the current API.